ush/ufsda/misc_utils.py

```diff
@@ -19,7 +19,7 @@ def calc_fcst_steps(fcst_step, win_length):
     """
     # need to get +- half of the window length
     # assumes only hours for now, probably bad...
-    h = re.findall('PT(\\d+)H', win_length)[0]
+    h = int(re.findall('PT(\\d+)H', win_length)[0])
     start = f'PT{h/2}H'
     end = f'PT{h*2}H'
```

Comment:
I believe `h` is already int type here, but `h/2` turns float. To keep it as int type, I use `h//2` and so the window would go from `h-h//2` to `h+h//2`.

Reply:
it is string before I explicitly make it `int` but good point about checking to make sure it is still an int after the division.

Reply:
Ah, you are right. I just tested, it is string and needs to be converted to int first.